x-pack/test/api_integration/apis/synthetics/sync_global_params.ts

```diff
@@ -0,0 +1,185 @@
+          geo: { lat: 0, lon: 0 },
+          url: 'mockDevUrl',
+          isServiceManaged: true,
+          status: 'experimental',
+          isInvalid: false,
+        },
+        {
+          concurrentMonitors: 1,
+          id: testFleetPolicyID,
+          isInvalid: false,
+          isServiceManaged: false,
+          label: 'Test private location 0',
+          geo: {
+            lat: '',
+            lon: '',
+          },
+          agentPolicyId: testFleetPolicyID,
+        },
+      ]);
+    });
+
+    it('adds a monitor in private location', async () => {
+      const newMonitor = browserMonitorJson;
+
+      newMonitor.locations.push({
+        id: testFleetPolicyID,
+        label: 'Test private location 0',
+        isServiceManaged: false,
+      });
+
+      const apiResponse = await supertestAPI
+        .post(API_URLS.SYNTHETICS_MONITORS)
+        .set('kbn-xsrf', 'true')
+        .send(newMonitor);
+
+      expect(apiResponse.body.attributes).eql(
+        omit(
+          {
+            ...newMonitor,
+            [ConfigKey.MONITOR_QUERY_ID]: apiResponse.body.id,
+            [ConfigKey.CONFIG_ID]: apiResponse.body.id,
+          },
+          secretKeys
+        )
+      );
+      newMonitorId = apiResponse.body.id;
+    });
+
+    it('added an integration for previously added monitor', async () => {
+      const apiResponse = await supertestAPI.get(
+        '/api/fleet/package_policies?page=1&perPage=2000&kuery=ingest-package-policies.package.name%3A%20synthetics'
+      );
+
+      const packagePolicy = apiResponse.body.items.find(
+        (pkgPolicy: PackagePolicy) =>
+          pkgPolicy.id === newMonitorId + '-' + testFleetPolicyID + '-default'
+      );
+
+      expect(packagePolicy.policy_id).eql(testFleetPolicyID);
+
+      comparePolicies(
+        packagePolicy,
+        getTestBrowserSyntheticsPolicy({ name: browserMonitorJson.name, id: newMonitorId })
+      );
+    });
+
+    it('adds a test param', async () => {
+      const apiResponse = await supertestAPI
+        .post(SYNTHETICS_API_URLS.PARAMS)
+        .set('kbn-xsrf', 'true')
+        .send({ key: 'test', value: 'test' });
+
+      expect(apiResponse.status).eql(200);
+    });
+    it('get list of params', async () => {
+      const apiResponse = await supertestAPI
+        .get(SYNTHETICS_API_URLS.PARAMS)
+        .set('kbn-xsrf', 'true')
+        .send({ key: 'test', value: 'test' });
+
+      expect(apiResponse.status).eql(200);
+
+      apiResponse.body.data.forEach(({ attributes }: SavedObject<SyntheticsParam>) => {
+        params[attributes.key] = attributes.value;
+      });
+    });
+
+    it('sync global params', async () => {
+      const apiResponse = await supertestAPI
+        .get(SYNTHETICS_API_URLS.SYNC_GLOBAL_PARAMS)
+        .set('kbn-xsrf', 'true')
+        .send({ key: 'test', value: 'test' });
+
+      expect(apiResponse.status).eql(200);
+    });
+
+    it('added params to for previously added integration', async () => {
```

Comment:
Can you also just add a test for deleting params.

Reply:
Done !!